AnimatedTextInput/Classes/AnimatedTextInputFieldConfigurator.swift

```diff
@@ -38,25 +44,41 @@ private struct AnimatedTextInputTextConfigurator {
     }
 }
 
+private struct AnimatedTextInputEmailConfigurator {
+    
+    static func generate() -> TextInput {
+        let textField = AnimatedTextField()
+        textField.keyboardType = .EmailAddress
+        textField.clearButtonMode = .WhileEditing
+        textField.autocapitalizationType = .None
+        textField.spellCheckingType = .No
+        textField.autocorrectionType = .No
+        
+        return textField
+    }
+}
+
 private struct AnimatedTextInputPasswordConfigurator {
 
-    static func generate() -> TextInput {
+    static func generateWithDisclosure(visible: Bool = false) -> TextInput {
         let textField = AnimatedTextField()
```

Comment:
If the default for visibility is false, you could call this as `generateWithDisclosure()` and not get a disclosure button, which is confusing.